boards/fruitbar/main.py

```diff
@@ -0,0 +1,284 @@
+import board
+
+from kb import KMKKeyboard
+
+from kmk.extensions.peg_oled_Display import (
```

Comment:
We recommend you use the `display` extension instead of `peg_oled`.

Reply:
I will give it a try.

Reply:
Ok I don't know what to do here.  I used peg_oled as an example and it works.  If I change it, what board do you suggest to use as a model?